README.md

```diff
@@ -102,20 +101,7 @@ For more information see [DataLoader](https://graphql-dotnet.github.io/docs/guid
 
 > *Note: Prior to version 4, the contents of this package was part of the main GraphQL.NET package.*
 
-#### 5. Subscriptions
```

Comment:
I would keep this section in changed way:
```
#### 5. Subscriptions

`DocumentExecuter` can handle subscriptions as well as queries and mutations.
For more information see [Subscriptions](https://graphql-dotnet.github.io/docs/getting-started/subscriptions).
```

Reply:
Done as suggested.